test/unit/store/cart-store.js

```diff
@@ -21,7 +21,7 @@ describe('CartStore', () => {
         Dispatcher.handleAction('STUDENT_FOUND', {
             id: '123456',
             name: 'Pope Francis',
-            itemAddresses: []
+            items: []
```

Comment:
I just tried and I can't even get the student to appear.  I don't feel like this change is causing that, but maybe?